files/materialized_view_for_lanes.sql

```diff
@@ -44,7 +45,8 @@ as
   WHERE works.presentation_ready = true
     AND works.simple_opds_entry IS NOT NULL
 
-  ORDER BY (editions.sort_title, editions.sort_author, licensepools.availability_time);
+  ORDER BY (editions.sort_title, editions.sort_author, licensepools.availability_time)
+  WITH NO DATA;
```

Comment:
This improves initial setup because when we create the indexes there's no data in the materialized view.